tutorials/cp-trial-quick-onboarding/cp-trial-quick-onboarding.md

```diff
@@ -22,29 +22,20 @@ primary_tag: products>sap-cloud-platform
 
 ---
 
-[ACCORDION-BEGIN [Step 1: ](Introduction to SAP Cloud Platform trial)]
+[ACCORDION-BEGIN [Step: ](Welcome to SAP Cloud Platform Trial)]
 
-So you've signed up for a trial on SAP Cloud Platform but are not quite sure where to start? This tutorial is here to help! Let's first learn a bit about what SAP Cloud Platform is and what you should expect from your trial.
-
-SAP Cloud Platform is an enterprise platform-as-a-service (enterprise PaaS) that provides application development services and capabilities which let you build, extend, and integrate business applications in the cloud.
+So you've signed up for a trial on SAP Cloud Platform but are not quite sure where to start? This tutorial is here to help!
 
 A trial account on SAP Cloud Platform enables you to experience it for free, offering access to a comprehensive set of platform services, as well as early access to beta functionality.
 
-Trial accounts are intended for personal exploration, and not for production use or team development. They allow restricted use of the platform resources and services:
-
- | Cloud Foundry Trial
----|---
-**Data centers** | Runs on third-party data centers from AWS, Azure and Google Cloud Platform
-**Trial duration** | Limited to 90 days
-**Strengths** | Allows you to use multiple programming languages such as Java, Node.js and community/bring-your-own language options
+Trial accounts are intended for personal exploration, and not for production use or team development. They allow restricted use of the platform resources and services.
 
-You can find the regions available on trial [here](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/350356d1dc314d3199dca15bd2ab9b0e.html) and then you can look for those regions in [this table](https://help.sap.com/doc/aa1ccd10da6c4337aa737df2ead1855b/Cloud/en-US/3b642f68227b4b1398d2ce1a5351389a.html) to see what services are available in trial.
 
 [DONE]
 [ACCORDION-END]
 
 
-[ACCORDION-BEGIN [Step 2: ](Global accounts, subaccounts, organizations and spaces)]
+[ACCORDION-BEGIN [Step: ](Global accounts, subaccounts, organizations and spaces)]
```

Comment:
This is actually my bad, but we should replace "organization" with "org" (not just here, all instacnces of it). That's the actual technical term in Cloud Foundry.